### ISSUE TITLE
Usage of useCucumberJVM property raises build error Configuration with name 'testCompile' not found

### ISSUE THREAD

**Issue by kshepitko** [allure-framework/allure-gradle#29]
[//]: # (
. Note: for support questions, please use Stackoverflow or Gitter**. 
. This repository's issues are reserved for feature requests and bug reports.
.
. In case of any problems with Allure Jenkins plugin** please use the following repository 
. to create an issue: https://github.com/jenkinsci/allure-plugin/issues
.
. Make sure you have a clear name for your issue. The name should start with a capital 
. letter and no dot is required in the end of the sentence. An example of good issue names:
.
. - The report is broken in IE11
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

#### I'm submitting a ... 
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?

Usage of useCucumberJVM property raises build error `Configuration with name 'testCompile' not found`

Excerpt from `build.gradle`:
```
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath "io.qameta.allure:allure-gradle:2.5"
    }
}
apply plugin: 'io.qameta.allure'
allure {
    version = '2.4.1'
    autoconfigure = true
    aspectjweaver = true
    allureJavaVersion = '2.0-BETA20'
    clean = true
    resultsDir = new File(project.getBuildDir(),  "all-allure-results")
// Causes errors if uncommented
//    useCucumberJVM {
//        version = '2.0-BETA20'
//    }
}
```

If `useCucumberJVM` closure definition is uncommented, it causes the following effect:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'my-tests'.
> Configuration with name 'testCompile' not found.
```

#### If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem

Use the code above to reproduce.

#### What is the expected behavior?

No errors during the build

#### Please tell us about your environment:

| Allure version | 2.0-BETA20 |
| --- | --- |
| Test framework | junit@4.12 |
| Allure adaptor | allure-cucumber-jvm@2.0-BETA20 |
| Generate report using | allure-gradle@2.5 |
| Gradle version | 4.1, 4.5 |
#### Other information 

[//]: # (
. e.g. detailed explanation, stacktraces, related issues, suggestions 
. how to fix, links for us to have more context, eg. Stackoverflow, Gitter etc
)

**Comment by vlsi** [comment]
Hi, I believe https://github.com/allure-framework/allure-gradle/pull/61 fixes the issue as the plugin no longer adds dependencies to `testCompile`.

Please feel free to try it and provide feedback

**Comment by vlsi** [comment]
Resolved in #61
